node-task-pkg/index.js

```diff
@@ -0,0 +1,51 @@
+const util = require("util");
+const { exec } = require("child_process");
+const execProm = util.promisify(exec);
+
+const os = require("os");
+const plat = os.platform();
+
+exports.getProcList = async function (username) {
+    let result = {stdout:"",stderr:""};
```

Comment:
I think we don't want to know how we get out and err outside. Maybe rename to `processes` and `error`?